IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory/ngsa-memory.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    name: ngsa
+  name: ngsa
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ngsa-memory
+  namespace: ngsa
```

Comment:
is the comment below necessary? If so, can we resolve it?

Reply:
Done